### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by blackbaud-conorwright**
Resolves: blackbaud/skyux2#2150
Docs: None

### CODE REVIEW

File: e2e/progress-indicator.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+import {
+  expect,
+  SkyHostBrowser
+} from '@skyux-sdk/e2e';
+
+describe('Progress indicator component', () => {
+  beforeEach(() => {
+    SkyHostBrowser.get('visual/progress-indicator');
+    SkyHostBrowser.setWindowBreakpoint('lg');
```
Comment (Blackbaud-SteveBrush) [inline e2e/progress-indicator.e2e-spec.ts:14]: Mind providing some `xs` screenshots too?

File: src/app/public/modules/progress-indicator/progress-indicator.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+      expect(buttonElements[0].disabled).toBeTruthy();
+
+      expect(fixture.componentInstance.navButtons.last.disabled).toBeTruthy();
+      expect(buttonElements[1].disabled).toBeTruthy();
+
+      expect(fixture.componentInstance.resetButton.disabled).toBeTruthy();
+      expect(resetButton.disabled).toBeTruthy();
+    }));
+  });
+});
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/progress-indicator/progress-indicator.component.spec.ts:416]: I'm not seeing any accessibility tests in here. Did they get moved over?

**Review by Blackbaud-AlexKingman — CHANGES_REQUESTED** [review]

File: src/app/public/modules/progress-indicator/progress-indicator.component.spec.ts (hunk 2 above)
Comment (blackbaud-conorwright) [reply src/app/public/modules/progress-indicator/progress-indicator.component.spec.ts:416]: missed it, added now though

File: e2e/progress-indicator.e2e-spec.ts (hunk 1 above)
Comment (blackbaud-conorwright) [reply e2e/progress-indicator.e2e-spec.ts:14]: done

**Comment by blackbaud-conorwright** [comment]
@Blackbaud-SteveBrush and @Blackbaud-AlexKingman 
should be set for review now. Turns out IE defaults to a transparent display for any styling on the component elements, so my background wasn't being drawn. I added display block to fix that

**Review by Blackbaud-AlexKingman — DISMISSED** [review]
LGTM.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**